Thai/Songs/ThaiSong_FanJa.md

```diff
@@ -14,3 +14,11 @@ Do you:
 [โทรหาเธอ](./callher/callher.md)
 
 [ลืมมันซะแล้วไปหาต้มยำกุ้งกิน](../Cuisine/TomYumKung.md)
+
+#####################################
+เว็บอะไรที่สร้างแรงบันดาลใจ หรือทำให้ชีวิตคุณดีขึ้น
+สำหรับผมขอเสนอเว็บสอนเกี่ยวกับการเงินส่วนบุคคล มีตั้งแต่พื้นฐาน จนถึงลงทุนในหุ้น  แบบมีหลักการจัดเต็ม  ทำโดยผู้ที่ไม่หวังสิ่งตอบแทนจริงๆ ครับ นอกจากส่งเสริมให้คนเรียนรู้อยู่ตลอดเวลา 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.